formats/specialist_document/frontend/examples/aaib-reports.json

```diff
@@ -0,0 +1,51 @@
+{
+	"public_updated_at" : "2015-07-09T08:17:10+00:00",
+	"format": "specialist_document",
+	"locale": "en",
+	"details" : {
+		"body" : "<h2 id=\"summary\">Summary:</h2>\n<p>The gyroplane began to move forward against the brakes before sufficient rotor rpm had been achieved for takeoff.  The pilot responded by re-positioning the control stick fully aft and the rotors struck the tailplane.  The pilot lost directional control and the right landing gear subsequently failed, causing the gyroplane to tip onto its right side.  The pilot was uninjured.</p>\n\n<h3 id=\"download-report\">Download report:</h3>\n<p><a rel=\"external\" href=\"https://assets.digital.cabinet-office.gov.uk/media/559e25de40f0b61564000039/Rotorsport_UK_Calidus_G-PCPC_07-15.pdf\">Rotorsport UK Calidus G-PCPC 07-15</a> </p>\n\n<h3 id=\"download-glossary-of-abbreviations\">Download glossary of abbreviations:</h3>\n<p><a href=\"https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/433812/Glossary_of_abbreviations.pdf\">Glossary of abbreviations</a></p>\n\n",
+		"metadata": {
+			"aircraft_category" : [
+				"sport-aviation-and-balloons"
+			],
+			"report_type" : "correspondence-investigation",
+			"date_of_occurrence" : "2015-08-08",
+			"aircraft_type" : "Rotorsport UK Calidus",
+			"location" : "Damyns Hall Aerodrome, Essex",
+			"registration" : "G-PCPC",
+			"document_type" : "aaib_report"
+		},
+		"headers" : [
+			{
+				"text" : "Summary:",
+				"level" : 2,
+				"id" : "summary",
+				"headers" : [
+					{
+						"text" : "Download report:",
+						"level" : 3,
+						"id" : "download-report",
+						"headers" : [ ]
```

Comment:
hmm, ok we could do that but it might mean some more bit of fiddling with the structured header extractor code to make it produce the right output https://github.com/alphagov/specialist-publisher/blob/52193f0b23986e7bc83c5ec2973e37d65f6b27f8/lib/specialist_document_header_extractor.rb#L15

I wonder if it's worth it?

any opinions @dsingleton @rboulton ?